recipes/itolapi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+    \nFrom Command Line\n~~~~~~~~~~~~~~~~~\n\n(If you would like to set any parameters other than the tree id,\nlocation to save the file, file format, and whether to display datasets,\nyou must use ItolExport\
+    \ from a Python program)\n\n::\n\n    $ itolexport.py TREEID FILELOCATION FORMAT [OPTIONS]\n\nOptions include:\n * ``-d``: show datasets\n * ``-v``: verbose output\n * ``-h``: help\n\nFrom Python\n\
+    ~~~~~~~~~~~\n\nRunning itolexport.py from a Python program allows you to use all the\noptions that iTOL has available.\n\n::\n\n    from itolapi import ItolExport\n    itol_exporter = ItolExport()\n\
+    \    itol_exporter.add_export_param_value('tree', tree_id)\n    assert format in ['png', 'svg', 'eps', 'ps', 'pdf', 'nexus', 'newick']\n    itol_exporter.add_export_param_value('format', format)\n \
+    \   itol_exporter.add_export_param_value(param_key, param_value)\n    itol_exporter.export(file_location)\n\nValid ``param_key`` and ``param_value`` values can be found on `the iTOL API page`_.\n\n\
+    Bugs/Comments\n-------------\nSend bugs and comments as issues on the  `Github`_ repository.\n\nDevelopment\n-----------\nTo run tests:\n\n::\n\n    python setup.py install\n    pip install -r requirements-test.txt\n\
+    \    mypy itolapi\n    coverage run setup.py test\n    coverage report -m\n\nTo update PyPI:\n\n::\n\n    pip install twine\n    python setup.py sdist bdist_wheel\n    twine upload dist/*\n\n.. _iTOL\
+    \ (Interactive Tree of Life): http://itol.embl.de/\n.. _iTOL website: http://itol.embl.de/\n.. _iTOL options: http://itol.embl.de/help.cgi#batch\n.. _the iTOL API page: http://itol.embl.de/help.cgi#batch\n\
+    .. _Github: https://github.com/albertyw/itolapi/\n\n\n.. |PyPI| image:: https://img.shields.io/pypi/v/itolapi.svg\n   :target: https://pypi.python.org/pypi/itolapi/\n   :alt: Latest Version\n\n.. |Python\
+    \ Version| image:: https://img.shields.io/pypi/pyversions/itolapi.svg\n\n.. |License| image:: https://img.shields.io/pypi/l/itolapi.svg\n   :target: https://pypi.python.org/pypi/itolapi/\n   :alt: License\n\
+    \n\n.. |Codeship| image:: https://codeship.com/projects/d6470c00-c832-0132-4536-627bbcd2f5ed/status?branch=master\n    :target: https://codeship.com/projects/75058\n    :alt: Codeship\n\n.. |CodeCov|\
+    \ image:: https://codecov.io/github/albertyw/itolapi/coverage.svg?branch=master\n    :target: https://codecov.io/github/albertyw/itolapi?branch=master\n\n.. |Code Climate| image:: https://codeclimate.com/github/albertyw/itolapi/badges/gpa.svg\n\
+    \    :target: https://codeclimate.com/github/albertyw/itolapi\n    :alt: CodeClimate\n\n.. |Dependency Status| image:: https://pyup.io/repos/github/albertyw/itolapi/shield.svg\n   :target: https://pyup.io/repos/github/albertyw/itolapi/\n\
+    \n\n"
+  doc_url: ''
+  dev_url: ''
+
+extra:
+  recipe-maintainers: ''
```

Comment:
Add yourself if you feel like it (no obligation to) or remove this key (and the entire section) as well.

Reply:
okay, I put my name for now since I will need to maintain this recipe for other tools.

Reply:
Use your GitHub user name and use a list like this:
```
extra:
  recipe-maintainers:
    - cokelaer
```

Reply:
done